app/Console/Commands/OneOff/SeedScheduledJobs.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Console\Commands\OneOff;
+
+use App\Models\V2\ScheduledJobs\ReportReminderJob;
+use App\Models\V2\ScheduledJobs\SiteAndNurseryReminderJob;
+use App\Models\V2\ScheduledJobs\TaskDueJob;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class SeedScheduledJobs extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'one-off:seed-scheduled-jobs';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Creates the initial definition of scheduled jobs to replace what was in Kernel.php';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
```

Comment:
This task duplicates the scheduled jobs removed from `Kernel.php`. The assumption is that by the time we've generated the reports seeded here, we'll have the new UI in place to set new execution and report generation times and won't need to do it by Command, or manually in tinker any longer.